### PR TITLE
fix CheckPvpTalentByIndex nil index error

### PR DESCRIPTION
**Pull request by mrbuds**
check C_SpecializationInfo.GetPvpTalentSlotInfo doesn't return nil

### CODE REVIEW

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -491,13 +491,14 @@ end
 function WeakAuras.CheckPvpTalentByIndex(index)
   if (index <= 3) then
     local talentSlotInfo = C_SpecializationInfo.GetPvpTalentSlotInfo(1);
+    if not talentSlotInfo return false
     local checkTalentId = talentSlotInfo.availableTalentIDs[index];
     return talentSlotInfo.selectedTalentID == checkTalentId;
   else
     local checkTalentId = C_SpecializationInfo.GetPvpTalentSlotInfo(2).availableTalentIDs[index - 3];
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua]: The error message seems to point to line 497. Which is this line. Seems GetPvpTalentSlotInfo(2) returned nil here.

Comment (emptyrivers) [reply WeakAuras/Prototypes.lua]: Additionally, the added line is malformed - it should be `if not talentSlotInfo then return false end`.

Comment (mrbuds) [reply WeakAuras/Prototypes.lua]: i sent a patch to fix this

File: WeakAuras/Prototypes.lua [hunk 2]
```diff
@@ -491,13 +491,14 @@ end
 function WeakAuras.CheckPvpTalentByIndex(index)
   if (index <= 3) then
     local talentSlotInfo = C_SpecializationInfo.GetPvpTalentSlotInfo(1);
+    if not talentSlotInfo then return false end
     local checkTalentId = talentSlotInfo.availableTalentIDs[index];
     return talentSlotInfo.selectedTalentID == checkTalentId;
   else
     local checkTalentId = C_SpecializationInfo.GetPvpTalentSlotInfo(2).availableTalentIDs[index - 3];
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua]: This needs to be checked too.

**Pull request merged**